src/sql/workbench/services/serverGroup/browser/serverGroupDialog.ts

```diff
@@ -128,8 +128,10 @@ export class ServerGroupDialog extends Modal {
 				this.preventDefaultKeyboardEvent(event);
 				this.focusNext();
 			} else if (event.equals(KeyCode.RightArrow) || event.equals(KeyCode.LeftArrow)) {
-				this.preventDefaultKeyboardEvent(event);
-				this.focusNextColor(event.equals(KeyCode.RightArrow));
+				if (event.target === DOM.$('input.colorbox.server-group-color')) {
```

Comment:
Does this allow going through all the color boxes correctly? The query would only get the first element unless I'm missing something - but the target element may be any of them. 

also would suggest taking the time to make a const for the `server-group-color` class so we're more sure that this will continue to work even if the class name is changed on the actual element for some reason in the future.

Reply:
Yes, it does. The element is the server-colors container.